aiosqlite/tests/smoke.py

```diff
@@ -354,6 +354,31 @@ async def test_iterdump(self):
                 ],
             )
 
+    async def test_cursor_on_closed_connection(self):
+        db = await aiosqlite.connect(TEST_DB)
+
+        cursor = await db.execute("select 1, 2")
+        await db.close()
+        with self.assertRaises(Exception):
+            await cursor.fetchall()
+        with self.assertRaises(Exception):
+            await cursor.fetchall()
+
+    async def test_cursor_on_closed_connection_loop(self):
+        db = await aiosqlite.connect(TEST_DB)
+
+        cursor = await db.execute("select 1, 2")
+        tasks = []
+        for i in range(100):
+            if i == 50:
+                tasks.append(asyncio.ensure_future(db.close()))
+            tasks.append(asyncio.ensure_future(cursor.fetchall()))
+        for task in tasks:
+            try:
+                await task
+            except Exception:
+                pass
```

Comment:
Is there a way to make sure the tasks are returning (or throwing) the expected values/exceptions?